pydm/display_module.py

```diff
@@ -1,40 +1,247 @@
+import functools
```

Comment:
Module name probably should just be `pydm.display` or is there a reason for `display_module`?

Reply:
Historical name... we can switch it to be `display` and add an import to `display` in here for backwards compatibility? 🤷‍♂

Reply:
That would make sense, if you're willing!

Reply:
I don't like the name `display_module` either - this is just ancient legacy stuff.  Rename if you want!

Reply:
Addressed via 2e20aee